### PR TITLE
Flipping cards

### PR DESCRIPTION
**Pull request by imogenmisso**

### CODE REVIEW

File: SwiftCards/GameViewController.swift [hunk 1]
```diff
@@ -70,37 +89,36 @@ class GameViewController: UIViewController {
...
+        imageView.isUserInteractionEnabled = true
+        press.minimumPressDuration = 0.5
+        imageView.addGestureRecognizer(press)
+    }
 
     func makeDraggable(imageView: UIImageView) {
         let drag = UIPanGestureRecognizer(target: self, action: #selector(pan))
         imageView.isUserInteractionEnabled = true
-        if imageView.gestureRecognizers!.contains(drag) == false {
+        if !(imageView.gestureRecognizers!.contains {$0 is UIPanGestureRecognizer}) {
```
Comment (Caitlin-cooling) [inline SwiftCards/GameViewController.swift]: 🎉

File: SwiftCards/Model/Card.swift [hunk 2]
```diff
@@ -23,6 +24,12 @@ class Card: Equatable, Codable {
         self.xPosition = x
         self.yPosition = y
     }
+    func faceDown() {
```
Comment (Caitlin-cooling) [inline SwiftCards/Model/Card.swift]: 👍

File: SwiftCards/Model/Playarea.swift [hunk 3]
```diff
@@ -19,6 +19,10 @@ class Playarea: Equatable, Codable {
         }
         return card
     }
+    func bringCardToFront(_ card: Card) {
```
[the remark this answers is not in the document]
Comment (imogenmisso) [reply SwiftCards/Model/Playarea.swift]: Yep!

File: SwiftCards/GameViewController.swift [hunk 4]
```diff
@@ -70,37 +89,36 @@ class GameViewController: UIViewController {
 
         // update model if new position is valid
         let newOrigin = CGPoint(x: newX, y: newY)
-        if validPosition(newOrigin, image: touchedImage) {
-            let touchedCard = getCardObject(image: touchedImage)
-            touchedCard.setCoords(x: Float(newX), y: Float(newY))
+        guard validPosition(newOrigin, image: touchedImage) else {
```
Comment (IrinaSTA) [inline SwiftCards/GameViewController.swift]: This is something new - gotta checkout 'guard' - love it!!!

**Pull request merged**